pkgs/development/libraries/agda/agda-categories/default.nix

```diff
@@ -11,6 +11,10 @@ mkDerivation rec {
     sha256 = "1acb693ad2nrmnn6jxsyrlkc0di3kk2ksj2w9wnyfxrgvfsil7rn";
   };
 
+  postPatch = ''
```

Comment:
You could add a comment here referring to this MR telling us to remove this patch again once `agda-categories` is bumped. Then we can merge already now.

Reply:
Good idea, did so! (And tested again.) So this seems good to merge to me. Will remove the WIP label.